### ISSUE TITLE
	transcription factor TFIIIB complex subunit  	IPR030701

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-annotation#1523]
This subunit is directly involved in transcription from pol iii

currently maps to
GO:0006359	regulation of transcription from RNA polymerase III promoter

could it move up to 
transcription from RNA polymerase III promoter

**Comment by ValWood** [comment]
actually the problem might not be the InterPRO mapping, It might be the F-P link
Will check and feedback....

**Comment by EBI-Hsinyu** [comment]
Hello Val, 

Thank you for this. I had a look at a few entries that include TFIIIB specific subunits-beta" and Brf (Brf1), such as IPR030701 and its child entry IPR017174, and IPR00812 and its child entry IPR029529. Both IPR030701 and IPR017174 have the regulation term. I have replaced it with the term you suggested "GO:0006383 transcription from RNA polymerase III promoter". However, for IPR00812, I have to give a more general term "GO:0006351 transcription, DNA-templated" to reflect that fact it match both TFIIB and Brf1. However, I did give the term GO:0006383 to its child IPR029529 Brf1.  

So, both beta" and Brf1 subunits of TFIIIB should get this term GO:0006383 in our next release. 

Regards, 
Hsin-Yu
